frontends/verilog/verilog_frontend.cc

```diff
@@ -237,7 +238,8 @@ struct VerilogFrontend : public Frontend {
 		bool flag_defer = false;
 		bool flag_noblackbox = false;
 		bool flag_nowb = false;
-		std::map<std::string, std::string> defines_map;
+        define_map_t defines_map;
```

Comment:
Indenting

Reply:
Gah, Emacs let me down again! Fixed in latest push.